### ISSUE TITLE
Unable to use phpredis driver

### ISSUE THREAD

**Issue by mathewpeterson** [bernardphp/BernardBundle#10]
I am not able to use the phpredis driver. I am using the following versions: `bernard/bernard => 50bc62608f8ee981c42522045bd94ce07b5d6f49`, `bernard/bernard-bundle => bb0ced39d96d8d8c3c2f96642455eab075b352d9`, and `snc/redis-bundle => c07fde6155bce0647204fbd4f9b249df95cc2922`

Here are my config files:

``` yml
bernard_bernard:
    driver: phpredis
    serializer: simple
```

``` yml
snc_redis:
    clients:
        bernard:
            type: phpredis
            alias: bernard
            dsn: %redis_dsn%
            logging: false
            options:
                prefix: %redis_prefix%
```

There seems to be two issues, one that I can resolve, and the other I can't seem to figure out.

First the one that I can resolve:

SncRedisBundle restructured their client (https://github.com/snc/SncRedisBundle/commit/30565b8156faf9dbd32690ed0f4c639e4de9fa3c#diff-babf1c52f9bae7fffab1219e227de24eR21)

Basically, they no longer return a instance of `\Redis` and instead return an instance of `ClientInterface`

I can temporarily bypass to change the typehint in the driver (https://github.com/bernardphp/bernard/blob/master/src/Bernard/Driver/PhpRedisDriver.php#L17) in the `bernard/bernard` bundle.

Now to the second issue I can't seem to figure out:

When I try and run the app/console bernard:consume my-message-queue command, I instantly receive a redis exception:

```
  [RedisException]
  read error on connection
```

I'm open to suggestions on how to debug this.

**Comment by sbacelic** [comment]
```
 [RedisException]
  read error on connection

Exception trace:
 () at /vendor/bernard/bernard/src/Driver/PhpRedisDriver.php:69
 Redis->blPop() at /vendor/bernard/bernard/src/Driver/PhpRedisDriver.php:69
 Bernard\Driver\PhpRedisDriver->popMessage() at /vendor/bernard/bernard/src/Queue/PersistentQueue.php:95
```
I found this issue while debugging "read error on connection" exception when phpredis driver is used. SncRedisBundle sets **connection_timeout** to 5 by default. Since blpop duration is also 5 seconds, exception is thrown. You need to make **connection_timeout** > 5 to avoid exception in the worker (consumer). I hope this will save someone else some headache 😄

**Comment by acrobat** [comment]
Is this still an issue or can it be closed?
